analyzer/tests/unit/__init__.py

```diff
@@ -16,6 +16,8 @@
 REPO_ROOT = os.path.abspath(os.environ['REPO_ROOT'])
 PKG_ROOT = os.path.join(REPO_ROOT, 'build', 'CodeChecker')
 
+os.environ["CC_DATA_FILES_DIR"] = PKG_ROOT
```

Comment:
Because now the unit tests need to read some files from the config directory which is referred through this environment variable. Other tests (functional, web, etc.) have this variable too.